### PR TITLE
Update even_odd.py

### PR DESCRIPTION
**Pull request by iamshashankkashyap**
print(n*n) dala hai

### CODE REVIEW

File: even_odd.py [hunk 1]
```diff
@@ -3,7 +3,7 @@ def even(n):
     if(n%2==0):
         print("even")
 # Shashank odd func goes here
-print(n)
```
Comment (Saumya154) [inline even_odd.py]: remove this line
else all fine

Comment (Saumya154) [reply even_odd.py]: remove this line too

Comment (iamshashankkashyap) [reply even_odd.py]: ok

**Review by Saumya154 — APPROVED** [review]
okay

**Review by Saumya154 — APPROVED** [review]
done

**Pull request merged**